src/bidiMapper/domains/context/BrowsingContextImpl.ts

```diff
@@ -1012,6 +1014,99 @@ export class BrowsingContextImpl {
 
     await this.#cdpTarget.toggleNetworkIfNeeded(enableNetwork);
   }
+
+  async locateNodes(
+    params: BrowsingContext.LocateNodesParameters
+  ): Promise<BrowsingContext.LocateNodesResult> {
+    // TODO: test sandboxing.
+    const realm = await this.getOrCreateSandbox(params.sandbox);
```

Comment:
Nit: Let's do validation first and then any creation.

Reply:
good point. Done